src/precompilation.jl

```diff
@@ -5,25 +5,35 @@ if get(ENV, "PLOTS_PRECOMPILE", "true") == "true"
         n = length(_examples)
         imports = sizehint!(Expr[], n)
         examples = sizehint!(Expr[], 10n)
-        for i in setdiff(1:n, _backend_skips[:gr], _animation_examples)
+        for i in setdiff(1:n, _backend_skips[:gr])
             _examples[i].external && continue
             (imp = _examples[i].imports) === nothing || push!(imports, imp)
             func = gensym(string(i))
-            push!(examples, quote
-                $func() = begin  # evaluate each example in a local scope
-                    # @show $i  # debug
-                    $(_examples[i].exprs)
-                    if $i == 1  # only for one example
-                        fn = tempname()
-                        pl = current()
-                        gui(pl)
-                        Sys.iswindows() || savefig(pl, "$fn.png")
-                        Sys.iswindows() || savefig(pl, "$fn.pdf")
+            push!(
+                examples,
+                quote
+                    $func() = begin  # evaluate each example in a local scope
+                        # @show $i  # debug
+                        $(_examples[i].exprs)
+                        if $i == 1  # only for one example
+                            fn = tempname()
+                            pl = current()
+                            gui(pl)
+                            savefig(pl, "$fn.png")
+                            savefig(pl, "$fn.pdf")
+                        end
+                        nothing
                     end
-                    nothing
-                end
-                $func()
-            end)
+                    try
+                        # During precompilation on the Windows GitHub CI in Julia 1.8.2+,
+                        # can fail here due to GR potentially failing to write out the
+                        # temporary file. Pending a fix, swallow the error and continue.
+                        $func()
+                    catch
+                        @debug "Plots: Failed a trial save during precompilation"
+                    end
```

Comment:
```suggestion
                        $func()
                    catch e
                        if e isa SystemError && Sys.iswindows() && get(ENV, "CI", "false") == "true"
                            # During precompilation on the Windows GitHub CI in Julia 1.8.2+,
                            # can fail here due to GR potentially failing to write out the
                            # temporary file. Pending a fix, swallow the error and continue.
                        else
                            throw(e)
                        end
                        @debug "Plots: Failed a trial save during precompilation"
                    end
```

If the warning doesn't work as expected, then maybe silently ignore precompilation `SystemEror` on `windows` `ci`, else hard error.

Reply:
That's fair. We probably want `rethrow()` though?

Reply:
Ha, yes I took `rethrow(e)` note in https://docs.julialang.org/en/v1/base/base/#Base.rethrow too seriously.